lipton_custom/fixtures/custom_field.json

```diff
@@ -14,7 +14,7 @@
   "dt": "Batch",
```

Comment:
Need to export fixtures again. traceability_details not included in this file

Reply:
Left out due to a type. There was a missing ','.